jupyterlab_wipp_plugin_creator/jupyterlab_wipp_plugin_creator/handlers.py

```diff
@@ -0,0 +1,128 @@
+import json
+import os
+
+from shutil import copy2
+from wipp_client.wipp import gen_random_object_id
+
+from jupyter_server.base.handlers import APIHandler
+from jupyter_server.utils import url_path_join
+from .log import get_logger
+import tornado
+
+from jinja2 import Template
+
+logger = get_logger()
+# logger.setLevel(logging.INFO)
+
+class WippHandler(APIHandler):
+    @property
+    def wipp(self):
+        return self.settings["wipp-plugin-creator"]
+
+
+# Create wipp plugin based on user input
+class CreatePlugin(WippHandler):
+    @tornado.web.authenticated
+    def post(self):
+        """
+        POST request handler,
+
+        Creates a temp folder with random name,
+        Generates plugin.json, requirements.txt, dockerfile,
+        Copies files
+        Builds dockerimage,
+
+        Input format:
+            {
+                'formdata':
+                    {
+                    'name': '',
+                    'version': '',
+                    'title': '',
+                    'description': ''
+                    }
+                'filepaths':
+                    string[]
+            }
+
+        """
+
+        pwd = os.getcwd()
```

Comment:
This returns the working directory of the process. In case of Jupyter: user's home folder